src/index.js

```diff
@@ -46,47 +44,35 @@ module.exports = {
     'id-length': ['error', {
       exceptions: ['_', 'e', 'i']
     }],
+    'id-match': ['error', '^[_a-zA-Z][a-zA-Z0-9]*$|^[A-Z][A-Z0-9]*_[A-Z0-9]+$'],
```

Comment:
@rplopes that case should be covered by [no-underscore-dangle](http://eslint.org/docs/rules/no-underscore-dangle).